.github/workflows/deploy.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Commit to develop branch and merge to master branch
+        if: github.ref == 'refs/heads/master'
```

Comment:
Odd that this change is a part of this PR? Was it left out before for some reason?

Reply:
This error wasn't addressed before, because until now it didn't cause failed builds. I'll move this fix in separate PR